tasks/icinga2-web2-Debian.yml

```diff
@@ -4,8 +4,11 @@
     name:
       - php
       - php-gd
+      - php-curl
+      - php-imagick
```

Comment:
The docs at https://icinga.com/docs/icingaweb2/latest/doc/20-Advanced-Topics/#automating-the-installation-of-icinga-web-2 also mention php-intl and php-mbstring. 
If you look at the RedHat chunk you'll find I install, which is interesting, because I'm still missing curl and mbstring and haven't seen php-dom anywhere as a requirement, nor should php-ldap be there as a permanent extension (an optional yes).

```
yum:
    name:
      - rh-php72-php-ldap
      - rh-php72-php-intl
      - rh-php72-php-gd
      - rh-php72-php-dom
      - rh-php72-php-imagick
```